docs/engage/front-end-ui/platform-overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Platform Overview
+sidebar_position: 1
+---
+
+## Introduction
+
+The platform provides a modular and extensible infrastructure tailored for decentralized finance (DeFi) applications, with a particular focus on the tokenization, governance, and utilization of real-world assets (RWA). By combining critical components such as staking, governance, buybacks, and vesting within a single architecture, the platform enables efficient on-chain financial operations aligned with regulatory and community-driven frameworks.
+
+The Engage Toolkit consists of several integrated modules designed to streamline token utility and life cycle management while supporting decentralized governance and community incentives. Each module operates independently but can be used in combination to support more complex use cases, including liquidity provisioning, project fundraising, and structured token distribution.
+
+Key modules include:
+
+- **Staking** – incentivize and engage communities
+- **Governance** – on-chain proposal and voting systems
+- **Buybacks** – deflationary token mechanics
+- **Vesting** – structured token releases
+- **Admin Section** – manage system parameters and users
+
+These modules are accessible via a clean, responsive frontend, compatible across devices and optimized for MetaMask, Trust Wallet, and WalletConnect.
+
+## UI Technology Stack
+
+Defactor's frontend architecture is designed to deliver performance, security, and flexibility:
+
+- **Frontend Framework**: React (Next.js)
+- **Styling**: TailwindCSS + ShadCN UI
```

Comment:
I think this is not correct , take a look at tghe package.json file for forntend repos , we use material UI and a different tech stack